configs/projects/puppet-agent.rb

```diff
@@ -71,7 +70,8 @@
     end
   elsif platform.is_windows?
     # For windows, we need to ensure we are building for mingw not cygwin
-    host = "--host #{platform.platform_triple}"
+    platform_triple = platform.platform_triple
+    host = "--host #{platform_triple}"
```

Comment:
Oh, wait, so `host` is and was being set correctly. However, if we ever want to reference `platform_triple` in the same way solaris and aix are accessing it (i.e., `settings[:platform_triple]`), then this is how we do that. We're not currently using that variable anywhere other then `settings[:host]`. It seemed cleaner to set it now rather then wait until we need it.